string kind
string label
string state
bool publish
